content/blog/gatsby-multi-author/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: How to support multiple authors in Gatsby.js
+date: 2020-03-20
+author: franleplant
+description:
+  "Gatsby Starter Blog doesn't support multiple authors by default. Let's fix that!
+  And in the Journey we will uncover how to use the file system as source of dynamic data"
+tags:
+  - gatsbyjs
+  - javascript
+  - react
+  - filesystem
+  - yaml
+  - blog
+---
+
+## Introduction
+
+Gatsby.js blog starter comes with out of the box support for a single author blog.
+But what happens when you want to support multiple authors writing in your blog?
+
+Gatsby actually makes it very easy to do so but for people getting started with Gatsby
+it might be a bit of challenge putting all the parts together.
+
+This post hopes to hollistically show how to solve this problem.
+
+## Doing it
+
+The most high level interface that authors writing posts will use will
+still be the Front Matter yaml on top of each markdown file.
+
+By specifying an _author id_ in our Front Matter we are going to be able
+to retrieve the rest of the author data we have stored in our codebase.
+
+```markdown:title=content/blog/my-post/index.md
+---
+title: My Post!
+author: author1
+---
+```
+
+For this to work we need to setup Gatsby to:
+
+- use the filesystem as a valid source of data. In this case we used `src/data` but it can be anything you like.
+- use a yaml transformer because we will store our authors data inside a yaml but you could use JSON or any other format.
+- map Front Matter's `author` attribute to `src/data/author.yaml`.
+- notice that you probably need to `yarn add gatsby-source-filesystem gatsby-transformer-yaml`
+
+```javascript:title=gatsby-config.js
```

Comment:
Link to gatsby-config for context? 
https://www.gatsbyjs.org/docs/gatsby-config/#plugins

Reply:
done